testsuite/forge/src/backend/k8s/swarm.rs

```diff
@@ -287,6 +276,27 @@ pub async fn list_services(client: K8sClient) -> Result<Vec<KubeService>> {
     services.into_iter().map(KubeService::try_from).collect()
 }
 
+pub async fn get_validators(client: K8sClient) -> Result<HashMap<PeerId, K8sNode>> {
+    let services = list_services(client).await?;
+    services
+        .into_iter()
+        .filter(|s| s.name.contains(VALIDATOR_LB))
+        .map(|s| {
+            let node_id = parse_node_id(&s.name).expect("error to parse node id");
+            let node = K8sNode {
+                name: format!("val-{}", node_id),
+                peer_id: PeerId::random(),
```

Comment:
I just copied this from @zihaoccc's original swarm setup code. I'm not sure how to fetch the peer ID, or if there's a way to hard code it for consistency across all tests

Reply:
oh, @bmwill nice catch here, I think it should be my mistake forget to add TODO here since then we did not have ability to operate on cluster to fetch config. @rustielin I think node config does have peer_id when we do deployment, I can do a double check